Helm doc revisions (#33713)
sourcegraph/sourcegraph: doc/admin/install/kubernetes/helm.md

> ℹ️ This quickstart guide is useful to those already familiar with Helm who have a good understanding of how to use Helm in the environment they want to deploy into, and who just want to quickly deploy Sourcegraph with Helm with the default configuration. If this doesn't cover what you need to know, see the links above for platform-specific guides.

> To use the Helm chart, add the Sourcegraph helm repository (on the machine/client ___from___ which you're going to be installing the chart):

Reviewer: this feels wrong to me.

helm should only be installed on the client-side (your machine, your CI/CD pipeline)

the helm repository can only be added on the client-side, so `helm` cli can pick it up

when we run `helm install`, it will create an instance of the chart on the server (your k8s cluster)

Reviewer 2: Hm, isn't that what this text update is saying? Helm commands should be run on the client. The wording might be confusing though, it always is when you start thinking too hard about what does "from" mean. Maybe @emchap can give some feedback here.

Reviewer: I am definitely not good at English...so I will leave it to you all to decide.

> on the machine/client ___from___ which you're going to be installing the chart

When I am reading this, I thought, I need to install helm cli on the machine where the chart will be installed on, which is the Kubernetes cluster.

Reviewer 3: I tend to agree that the new text is confusing (though of course I also know Helm very minimally), but since @jwayman is officially the person who manages docs content I defer to him on best phrasing. :)

Reviewer 3: Given the nature of the review being requested here, he likely makes a more appropriate reviewer than I am, though happy to read through it once you settle on phrasing!

Author: Revised it to "on the machine used to interact with your cluster".

I think that's the clearest we're going to be able to get to without explaining a lot.

Reviewer 3: I think that feels reasonably clear to me. (Still defer to Jeff, though.)

1. You need to have a GKE cluster (>=1.19) with the `HTTP Load Balancing` addon enabled. Alternatively, you can use your own choice of Ingress Controller and disable the `HTTP Load Balancing` add-on, [learn more](https://cloud.google.com/kubernetes-engine/docs/how-to/custom-ingress-controller).
1. Your account should have sufficient access rights, equivalent to the `cluster-admin` ClusterRole.
1. Connect to your cluster (via either the console or the command line using `gcloud`) and ensure the cluster is up and running by running: `kubectl get nodes` (several `ready` nodes should be listed)

> 1. Have the [Helm CLI](https://helm.sh/docs/intro/install/) installed and run the following command to link to the Sourcegraph helm repository (on the machine/client ___from___ which you're going to be installing the chart):

Author: ```suggestion
1. Have the [Helm CLI](https://helm.sh/docs/intro/install/) installed and run the following command to link to the Sourcegraph helm repository (on the machine used to interact with your cluster):
```

> You may consider using your custom Ingress Controller instead of Application Gateway, [learn more](https://docs.microsoft.com/en-us/azure/aks/ingress-basic)
1. Your account should have sufficient access equivalent to the `cluster-admin` ClusterRole.
1. Connect to your cluster (via either the console or the command line using the Azure CLI) and ensure the cluster is up and running using: `kubectl get nodes` (several `ready` nodes should be listed)

> 1. Have the [Helm CLI](https://helm.sh/docs/intro/install/) installed and run the following command to link to the Sourcegraph helm repository (on the machine/client ___from___ which you're going to be installing the chart):

Author: ```suggestion
1. Have the [Helm CLI](https://helm.sh/docs/intro/install/) installed and run the following command to link to the Sourcegraph helm repository (on the machine used to interact with your cluster):
```
